common/religion/doctrines/wc_doctrines_religions.txt

```diff
@@ -102,12 +115,14 @@
 		parameters = {
 			hostility_same_religion = 1
 			hostility_same_family = 1
-			hostility_others = 3
+			hostility_others = 2
 			
 			#Syncretic religions
 			hostility_override_tenet_shadow_syncretism = 1
 			
 			#Pure evil religions
+			hostility_override_intolerance_hostility_doctrine = 3
+			doctrine_parameter_hostility_override_intolerance_hostility_doctrine:0 ""
```

Comment:
Wrong place. It must be added in loc file like `localization/english/religion/wc_core_tenets_l_english.yml`.

Reply:
done, moved them all to the loc file